lib/websockex.ex

```diff
@@ -410,19 +410,19 @@ defmodule WebSockex do
   error tuple with a `WebSockex.ConnError` exception struct as the second
   element.
   """
-  @spec send_frame(client, frame) ::
+  @spec send_frame(client, frame, timeout) ::
           :ok
           | {:error,
              %WebSockex.FrameEncodeError{}
              | %WebSockex.ConnError{}
              | %WebSockex.NotConnectedError{}
              | %WebSockex.InvalidFrameError{}}
           | none
-  def send_frame(client, _) when client == self() do
+  def send_frame(client, _, _) when client == self() do
     raise %WebSockex.CallingSelfError{function: :send_frame}
   end
 
-  def send_frame(client, frame) do
+  def send_frame(client, frame, timeout \\ 10000) do
     try do
       {:ok, res} = :gen.call(client, :"$websockex_send", frame)
```

Comment:
Looks like this isn't actually doing anything with the timeout?
```suggestion
      {:ok, res} = :gen.call(client, :"$websockex_send", frame, timeout)
```